templates/manager/email-instance.html

```diff
@@ -31,10 +24,28 @@ <h1 class="h2 heading-underline">{{instance.email.title}} Instance</h1>
       <strong>Note:</strong> This instance was cancelled manually. <span class="fas fa-info-circle" data-toggle="tooltip" data-placement="top" title="This instance was cancelled before emails were sent to all recipients."></span>
     </div>
   {% endif %}
-  <div class="row">
+  <div class="row align-content-center">
```

Comment:
I might be missing something, but I'm not sure if the `align-content-center` class is needed here?